README.md

```diff
@@ -79,3 +79,26 @@ members. These tools do not receive regular product updates or security updates.
 [rubysdk]: https://github.com/cburnette/boxr
 [chromeimg]: images/chrome.png
 [chromesdk]: https://github.com/box/Chrome-App-SDK
+
+# Sample code
+The following sections aggregate existing demo application built with Box SDKs or CLI and showcases the functionality. Projects that are built by and for the Box developer community
+
+You can reference it and use it as inspiration. Support for these projects is provided by the individual maintainers and might vary from one project to the next.
+
+For Box's official SDKs or CLI, and any other officially supported examples make sure to head over to official tool page and search for examples.
+
+## Developed by Box
+
+|  Link   | Description |
+|-----|--|
+|  [Box Java SDK Samples](https://github.com/box/box-java-sdk-samples)   | Includes example sample that can create Box App User account and leverages webhook and AWS service. |
+|    [Box Samples](https://github.com/box/samples) | Repo contains Box SDKs examples in Node, Java, Angular and .Net languages. |
+| [Collection of Box Samples](https://github.com/box-community) | List of various sample and demo applications |
```

Comment:
Maybe this should be in the community section? We are aggregating this but it is created by community

Reply:
updated